src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+	
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.BotService</PackageId>
+    <Description>Microsoft Azure Management Bot Service Library</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.BotService</AssemblyName>
+    <PackageTags>Microsoft Azure Bot Service management;Bot;Bot Service;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      This is a public release of the Azure Bot Service SDK. Included with this release is bot management and channel registration.
+      ]]>
+    </PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[2.28.3]" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="[3.13.9]" />
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="[5.1.2]" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Generated\BotServicesChannelOperations.cs" />
```

Comment:
What is the intention here?
@shahabhijeet PTAL

Reply:
I'm sorry, what specifically are you curious about? Is your questions around the different identity dependencies?

Reply:
I'm curious as to why these files are not included during the compilation